docs/config.md

```diff
@@ -123,7 +123,12 @@ For a good example, see https://develop.element.io/config.json.
   `/.well-known/matrix/client` in its well-known location, and the JSON file
   at that location has a key `m.vector.riot.jitsi`. In this case, the
   configuration found in the well-known location is used instead.
-
+1. `jitsiWidget`: Options to change the built-in Jitsi widget behaviour. `jitsi` controls
```

Comment:
Should this be called something generic so the same config can be applied to new voip solutions? Next gen voip will also have some sort of 'join conference' button I think

Reply:
I'd rather not until we know precisely what it looks like in-app. Migrations of config settings are relatively easy, too.